File: search_js.py
import re

with open('c:\\Users\\kollu\\EDCO\\app.js', 'r', encoding='utf-8') as f:
    content = f.read()

lines = content.splitlines()
patterns = [
    r'onAuthStateChanged',
    r'signInWith',
    r'handleNavigation',
    r'updateGlobalUserData',
    r'renderIdCard',
    r'withdraw',
    r'google',
    r'github'
]

for pat in patterns:
    print(f"=== Matches for {pat} ===")
    r = re.compile(pat, re.IGNORECASE)
    for idx, line in enumerate(lines):
        if r.search(line):
            print(f"{idx + 1}: {line.strip()}")

File: search_js.js
const fs = require('fs');
const content = fs.readFileSync('c:\\Users\\kollu\\EDCO\\app.js', 'utf8');
const lines = content.split(/\r?\n/);
const patterns = [
  /onAuthStateChanged/i,
  /signInWith/i,
  /handleNavigation/i,
  /updateGlobalUserData/i,
  /renderIdCard/i,
  /withdraw/i,
  /google/i,
  /github/i
];

patterns.forEach(pat => {
  console.log(`=== Matches for ${pat} ===`);
  lines.forEach((line, idx) => {
    if (pat.test(line)) {
      console.log(`${idx + 1}: ${line.trim()}`);
    }
  });
});
